docs/world_generation.md

```diff
@@ -0,0 +1,75 @@
+# World generation
+World generation in Hypermine is constrained the following principles:
+* The generated contents of each chunk must depend only on the parameters of the world generation algorithm, not on gameplay.
```

Comment:
It's not obvious to me what point this is trying to make. How could initial chunk contents depend on gameplay?